collector/collector.lua

```diff
@@ -10,6 +10,7 @@ require('tables')
 local slips = require('slips')
 local res_items = require('resources').items
 local key_items = require('resources').key_items
+local item_descriptions = require('resources').item_descriptions
```

Comment:
At this point there's no longer need to keep these tables separate and we might just as well have a single `res` table.

Reply:
Fixed in [691cb22](https://github.com/lili-ffxi/FFXI-Addons/pull/13/commits/691cb22ca38feac47d38f43db13cedd8f5834ad3)